src/style.css

```diff
@@ -118,28 +141,122 @@ h6 {
 }
 
 input {
-  border: 5px solid black;
+  border-image: linear-gradient(to right bottom, #c53306, #dc3acc);
+  border-width: 3px;
+  border-style: solid;
+  border-image-slice: 1;
+  height: 50px;
+  width: 250px;
+}
+
+/* remove arrows from number input field */
+
+/* Chrome, Safari, Edge, Opera */
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+/* Firefox */
+input[type=number] {
+  -moz-appearance: textfield;
 }
 
 .table {
-  border: 5px solid black;
+  /* border: 3px solid orangered; */
   width: 300px;
-  height: 200px;
+  min-height: 40px;
+  border-image: linear-gradient(to right bottom, #260b3c, #a053df);
+  border-width: 3px;
+  border-style: solid;
+  border-image-slice: 1;
 }
 
 tr {
   height: 10px;
 }
 
 tr:nth-child(odd) {
-  background-color: rgb(234, 230, 230);
+  background-color: #b16de88a;
 }
 
```

Comment:
- Kindly change the border color to black as it shows in the design provided it will make your design much more to the design provided as it was required in the project :+1: 
- Kindly make sure you use the appropriate color for the background I can suggest you use `tr:nth-child(odd) {
    background-color: #c8cfe0f0; }` 


## your design 

![image](https://user-images.githubusercontent.com/95297251/204087707-70b3fa4c-1e12-403b-9aa5-b1cbad428373.png)

## design provides 

![image](https://user-images.githubusercontent.com/95297251/204087727-2229db8a-c535-483b-b452-5d919180e303.png)